Content.Server/Sticky/Systems/StickySystem.cs

```diff
@@ -20,17 +21,48 @@ public sealed class StickySystem : EntitySystem
     [Dependency] private readonly SharedHandsSystem _handsSystem = default!;
     [Dependency] private readonly SharedInteractionSystem _interactionSystem = default!;
     [Dependency] private readonly SharedAppearanceSystem _appearance = default!;
+    [Dependency] private readonly EntityLookupSystem _lookup = default!;
 
     private const string StickerSlotId = "stickers_container";
 
     public override void Initialize()
     {
         base.Initialize();
+        SubscribeLocalEvent<TransferStickySurfaceOnSpawnBehaviorComponent, OnSpawnFromSpawnEntitiesBehaviourEvent>(TransferStickySurfaceFromDestroyedEntity);
+
+        SubscribeLocalEvent<StickyComponent, MapInitEvent>(OnMapInit);
         SubscribeLocalEvent<StickyComponent, StickyDoAfterEvent>(OnStickFinished);
         SubscribeLocalEvent<StickyComponent, AfterInteractEvent>(OnAfterInteract);
         SubscribeLocalEvent<StickyComponent, GetVerbsEvent<Verb>>(AddUnstickVerb);
     }
 
+    private void OnMapInit(EntityUid uid, StickyComponent component, MapInitEvent args)
+    {
+        if (!component.StickOnStart || component.StuckTo != null)
+            return;
+        
+        var stuck = false;
+        foreach (var entity in _lookup.GetEntitiesIntersecting(Transform(uid).MapPosition))
+        {
+            if (TryStickToEntity(uid, entity, null, component))
+            {
+                component.StickOnStart = false;
+                stuck = true;
+                break;
+            }
+        }
+
+        if (!stuck)
+        {
+            // entity was unable to stick on anything; draw it as unsticked
+            if (TryComp(uid, out AppearanceComponent? appearance))
+            {
+                _appearance.SetData(uid, StickyVisuals.IsStuck, false, appearance);
```

Comment:
SetData would check for appearance by itself, so getting it here is not necessary since its not used again in the method